src/cndict_loader.c

```diff
@@ -6,7 +6,7 @@
  * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
  * GNU Affero General Public License v3 (AGPLv3).
 */
-#include "buffer.h"
+#include "buffer/buffer.h"
```

Comment:
These should not be needed because you have `${root}/src/buffer` in the include directories in the CMakeLIsts file.

https://github.com/RediSearch/RediSearch/pull/6378/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR211

I know the Rust ffi crate will fail to build. Just add the new header location to its includes section as well

https://github.com/RediSearch/RediSearch/blob/b2b8e5c2c734b94ab87ec661f614a9c1467c902a/src/redisearch_rs/ffi/build.rs#L32-L49